### PR TITLE
[TC2-1254] Colombia documentation

### PR DESCRIPTION
**Pull request by kalmerkaurson**

### CODE REVIEW

File: affiliates/colombia.swagger.json [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
...redy.eu/v3/)\n\nStaging URL: [http://api.staging.credy.eu/v3/](http://api.staging.credy.eu/v3/)\n\nAPI supports post fields, XML and JSON.\n\nTo get response error messages in Colombian, \"Accept-Language\" header’s value must be set to according ISO code. Ex: `id` or `id-ID`.\n\n## Authorization\nAuthorization is performed with OAuth 2.0 Client Credentials Grant.\nWhen requesting token, the client key and secret key must be sent in request body.\nClient key and secret key will be assigned to you by Credy.\nThe requests to api must be authenticated with [bearer token](https://tools.ietf.org/html/rfc6749#section-7.1).\n\n## Lead registration flow\n\n1. Request is sent to /v3/customers\n\n2. If request is successful, customer uuid must be stored\n\n3. Request is sent to /v3/leads with previously stored customer uuid in query parameter \"customer\" *ex. /v3/leads?customer=a3ff7c20-e9e8-43a0-bfb7-71a5cf43b8d3* \n4. When the lead is accepted by api (under_investigation is received from POST lead), please redirect the customer to [https://www.credy.co/loading/?lead={UUID}](https://www.credy.co/loading/?lead={UUID}) (where {UUID} is replaced by the uuid that you receive from POST lead)\n"
```
Comment (Nuffic) [inline affiliates/colombia.swagger.json]: credy.co is not our site - please change to credy.com.co

Comment (Nuffic) [inline affiliates/colombia.swagger.json]: wrong language code

File: affiliates/colombia.swagger.json [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+            "type" : "boolean",
+            "example" : true
+          },
+          "terms_of_service" : {
+            "type" : "boolean",
+            "example" : true
+          }
+        }
+      },
+      "Customer" : {
```
Comment (Nuffic) [inline affiliates/colombia.swagger.json:204]: Can you list only the fields that we have on https://www.credy.com.co/registro-prestamo-corto/#contact ? For example exmployer or employer_phone is not even asked on the form, so not worth a mention.

File: affiliates/colombia.swagger.json [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+            "format" : "int32",
+            "description" : "Term for credit requested",
+            "example" : 30
+          },
+          "ip_address" : {
+            "type" : "string",
+            "description" : "IP address of customer",
+            "example" : "192.168.0.255"
+          },
+          "loan_purpose" : {
```
Comment (Nuffic) [inline affiliates/colombia.swagger.json]: not used in CO

File: affiliates/colombia.swagger.json (hunk 1 above)
Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json]: fixed

Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json]: is co and co-CO correct?

File: affiliates/colombia.swagger.json (hunk 2 above)
Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json:204]: removed unnecessary fields

File: affiliates/colombia.swagger.json (hunk 3 above)
Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json]: removed

File: affiliates/colombia.swagger.json (hunk 1 above)
Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json]: es-CO seems to be correct according to ISO-639

File: affiliates/colombia.swagger.json [hunk 4]
```diff
@@ -0,0 +1,410 @@
+{
+  "openapi" : "3.0.0",
+  "info" : {
+    "version" : "3.0.0",
+    "title" : "Colombia docs",
```
Comment (mikk150) [inline affiliates/colombia.swagger.json]: should be "Colombia affiliates API"

File: affiliates/colombia.swagger.json [hunk 5]
```diff
@@ -0,0 +1,410 @@
...
+          "phone" : {
+            "type" : "string",
+            "example" : "+57 305-3070600"
+          },
+          "address" : {
+            "$ref" : "#/components/schemas/PrimaryAddress"
+          },
+          "occupation" : {
+            "type" : "string",
+            "enum" : [ "EMPLOYED_INDEFINITE_PERIOD", "EMPLOYED_SPECIFIED_PERIOD", "WRITTEN_CONTRACT_OR_ORDER", "ECONOMIC_ACTIVITY", "STUDENT", "MATERNITY_LEAVE", "PENSIONER1", "BENEFITS", "FARMER", "UNEMPLOYED", "OTHER" ],
```
Comment (mikk150) [inline affiliates/colombia.swagger.json]: I counted to 8, you have 11...

**Review by mikk150 — CHANGES_REQUESTED** [review]

File: affiliates/colombia.swagger.json (hunk 4 above)
Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json]: fixed

File: affiliates/colombia.swagger.json (hunk 5 above)
Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json]: fixed

File: affiliates/colombia.swagger.json (hunk 4 above)
Comment (kalmerkaurson) [reply affiliates/colombia.swagger.json]: other countries have "Countryname docs" should I change it there as well?

Comment (mikk150) [reply affiliates/colombia.swagger.json]: https://documentation.credy.com/docs/mexico-affiliates

But no, not right now

**Review by mikk150 — APPROVED** [review]

**Pull request merged**